samples/deploy_and_test.py

```diff
@@ -325,16 +327,18 @@ def test_sayHello_ias(self):
 
         resp = self.perform_get_request_with_ias_token('/v1/sayHello', ias_service.fetch_ias_token(self))
         self.assertEqual(resp.status, 403, 'Expected HTTP status 403')
-        self.prompt_user_role_assignment()
-        logging.warning("In case after adding role collection, user is still not authorized. "
-                        "Check in IAS admin panel that the application's '{}' Subject Name Identifier is set to email. "
-                        "Bug: https://jtrack.wdf.sap.corp/browse/NGPBUG-139441 "
-                        .format(ias_service.ias_service_name))
-        resp = self.perform_get_request_with_ias_token('/v1/sayHello', ias_service.fetch_ias_token(self))
-        self.assertEqual(resp.status, 200, 'Expected HTTP status 200')
-        xsappname = self.get_deployed_app().get_credentials_property('xsappname')
-        self.assertRegex(resp.body, xsappname, 'Expected to find xsappname in response')
-
+        if (self.prompt_user_role_assignment()):
+            resp = self.perform_get_request_with_ias_token('/v1/sayHello', ias_service.fetch_ias_token(self))
+            if(resp.status != 200):
+                logging.warning("In case after adding role collection, user is still not authorized. "
+                                "Check in IAS admin panel that the application's '{}' Subject Name Identifier is set to email. "
+                                "Bug: NGPBUG-139441 "
+                                .format(ias_service.ias_service_name))
+            self.assertEqual(resp.status, 200, 'Expected HTTP status 200')
+            xsappname = self.get_deployed_app().get_credentials_property('xsappname')
+            self.assertRegex(resp.body, xsappname, 'Expected to find xsappname in response')
+        else:
+            logging.warning('test_sayHello_ias was skipped. Enable it with USER_INPUT_ENABLED=true.')
```

Comment:
@nenaraab I think it would be also good to mention in warning message to enter y instead of n for the first input. Something like 'test_sayHello_ias was skipped. To run test enable it with USER_INPUT_ENABLED=true and enter 'y' for the user prompt 'Do you want to add role manually?'

Reply:
yeah.. logging.warning('test_sayHello_ias was skipped. Enable it with USER_INPUT_ENABLED=true.')